nameable.rb

```diff
@@ -0,0 +1,5 @@
+class Nameable
+  def correct_name
+    raise NotImplementedError
+  end
+end
```

Comment:
- [ **OPTIONAL** ] Good job creating method 'correct_name' raising an error. However to make it be more explicit, I suggest you add a the `method` and `class` in the error message. This will ensure that user knows which method or class has an error. I can look similar to this:
```ruby

   ...
   def correct_name
        raise NotImplementedError, "#{self.class} has not implemented method '#{__method__}'"
    end
```